input/fsh/profiles.fsh

```diff
@@ -0,0 +1,14 @@
+Profile: ncpi-disease
+Parent: Condition
+Id: ncpi-disease
+Title: "NCPI Project Forge Disease"
+* ^version = "0.1.0"
+* ^status = #draft
+
+Profile: ncpi-phenotype
+Parent: Condition
+Id: ncpi-phenotype
+Title: "NCPI Project Forge Phenotype"
+* ^version = "0.1.0"
+* ^status = #draft
+
```

Comment:
I see you've created this fsh file to curate the custom profiles. Let's talk about this further.